routes/admin.js

```diff
@@ -10,8 +12,10 @@ router.get("/add", (req, res, next) => {
     res.render("problem_add");
 });
 
-router.get("/edit/:qID", (req, res, next) => {
-    res.render("problem_edit");
+router.get("/edit/:qID",async (req, res, next) => {
+    const ques = await Question.findOne({"qID":req.params.qID});
+    const t_case = await TC.findOne({"qID":req.params.qID});
+    res.render("problem_edit",{ques,t_case});
 });
```

Comment:
Put this in a function `editQuestion` which is in `./controls/admin.js`

Reply:
these have to be passed to the page before rendering, so I'd have to create a different route just to pass the problem statement and testcase.
It is more readable this way

Reply:
Oh! My bad. I thought it was `router.put`. You can then put this in a function `helper.getQuestion = async (req, res, next) =>{...}` and then the `admin.js` will look clean: `router.get("/edit/:qID", admin.getQuestion);`